website/content/v1.5/reference/configuration.md

```diff
@@ -583,14 +591,11 @@ scheduler:
 {{< /highlight >}}</details> | |
 |`discovery` |<a href="#clusterdiscoveryconfig">ClusterDiscoveryConfig</a> |Configures cluster member discovery. <details><summary>Show example(s)</summary>{{< highlight yaml >}}
 discovery:
-    enabled: true # Enable the cluster membership discovery feature.
```

Comment:
seems we lost all comments?

Reply:
omg, I need to look into that, that was a change to use pointer receivers, but I can figure out some other way to fix that